behaviour/graql/language/define.feature

```diff
@@ -664,6 +725,55 @@ Feature: Graql Define Query
       | REC |
 
 
+  Scenario: define `sub role` creates a role, provided it is used in a relation
+    Given graql define
+      """
+      define
+      team-member sub role;
```

Comment:
ok we can keep this for now, but in 1.9 i think this won't be possible anymore :)

Reply:
I see that as a good thing - means (some of) our 1.9 changes are documented in BDD